mod_app/utils/mixins.py

```diff
@@ -25,10 +25,16 @@ def file_preview(self, obj):
         )
 
     def link_preview(self, obj):
-        return format_html(
-            '<img src="{}" style="max-width: 15rem;" />',
-            obj.url,
-        )
+        if obj.__class__.__name__ == "Video":
```

Comment:
```suggestion
        if isinstance(obj, Video):
```
would it make sense to do this instead (and add `from mod_app.models.support_models import Video` to the imports? Or perhaps `obj._meta.verbose_name == "Video"`?

Reply:
I'll try it! I know I ran into circular import issues before but it might not be the case with this file

Reply:
The `isinstance` didn't seem to work when I tested it, is there a particular benefit to using `obj._meta.verbose_name == "Video"` instead of the `obj.__class__.__name__`?